services/federator/src/Federator/Remote.hs

```diff
@@ -63,10 +63,10 @@ data RemoteError
   deriving (Show)
 
 instance AsWai RemoteError where
-  toWai (RemoteError _ _ e) =
-    federationRemoteHTTP2Error e
-  toWai (RemoteErrorResponse _ _ status body) =
-    federationRemoteResponseError status body
+  toWai (RemoteError target msg err) =
+    federationRemoteHTTP2Error (cs $ show (target, msg)) err
```

Comment:
The error type contains `errorData` we should use that instead of stringifying everything. If we really want to stringify this, can we use something better than a tuple?